### PR TITLE
App refactoring

### PR DESCRIPTION
**Pull request by SergSkol**
## Refactored app using classes
- Change object to class;
- Change code in removeBook method to filter;
- Adjust load from local storage;

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -2,87 +2,75 @@ const storageKey = 'awesomeBooks';
...
-  };
-  arrBooks.push(newBook);
-}
+  addBook(newTitle, newAuthor) {
+    const timeNow = new Date();
+    const newBook = {
+      id: timeNow.getTime(),
+      title: newTitle,
+      author: newAuthor,
+    };
```
Comment (julie-ify) [inline app.js:18]: - [ ] I am unable to add a book. After populating inputs and clicking the add button, the app does not save the book on the local storage and neither does it present it on the homepage. Kindly fix this.

  absg

File: app.js [hunk 2]
```diff
@@ -2,87 +2,75 @@ const storageKey = 'awesomeBooks';
...
 
-function addElement(elementType, parent, className) {
-  const element = document.createElement(elementType);
-  element.classList.add(className);
-  parent.appendChild(element);
-  return element;
-}
+  removeBook(id) {
+    this.arrBooks = this.arrBooks.filter((book) => book.id !== id);
+  }
```
Comment (julie-ify) [inline app.js:24]: - [ ] Since I was unable to add a book I was unable to verify that the book removal function works. Kindly ensure that as well.

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @SergSkol, @DiamondNath07 and @atatm,

Good job so far!

## Highlights!
- Good feature branch name ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
## - Kindly update your readme using the template as provided by microverse

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by m05e5 — CHANGES_REQUESTED** [review]
Hi @SergSkol, @atatm  and @DiamondNath07  

Good job so far. !
There are some issues that you still need to work on to go to the next project but you are almost there!
mario_ladders


## Highlights


## Required Changes ♻️
* You really did a great job but there are some features you need to improve

- [ ] as mentioned by the previous reviewer, Kindly update your readme using the [README template](https://github.com/microverseinc/readme-template/blob/master/README.md) provided by microverse

_I'm sure you can do it just keep it up_

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_@m05e5_

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by emmiiorji — APPROVED** [review]
Hi @SergSkol, 

# Status: Approved ✅

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Red-approved

## Good Highlights

- Linter checks are passing
- Adherence to GitHub flow
- Requested changes have been implemented
- Good commit messages


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**